beachball.config.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ * @ts-check
+ */
+
+const {execSync} = require('child_process');
+
+module.exports = {
+  ...require('@rnw-scripts/generated-beachball-config'),
+  
+  // Do not generate tags for monorepo packages by default, to avoid a GitHub
+  // release for every package.
+  gitTags: false,
+
+  // Default commit message, used if we do not detect an RNW version update
+  message: 'Publish Packages | ***NO_CI***',
+
+  hooks: {
+    // Stamp versions when we publish a new package
+    prepublish: (_packagePath, name, version) => {
+      if (name === 'react-native-windows') {
+        console.log(`Stamping RNW Version ${version}`);
+
+        execSync(`yarn stamp-version ${version}`);
+
+        // prepublish is run before bumping or commiting. Append an additional
+        // commit to check-in stamped versions when we push.
+        execSilent('git stash', );
+        execSilent('git add --all');
+        execSilent(`git commit --message "Stamp RNW ${version}"`);
```

Comment:
This will end up producing two commits, with the beachball one (now in beachball.config.js) coming after this one. More spam commits is a little annoying, but the only hook is before the beachball commit is added (which reverts local changes).

Reply:
So, something like:
1. `Stamp RNW 0.0.0-canary.345`
2. `Publish Packages | ***NO_CI***`

with 2 being the latest